platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -19,6 +19,17 @@ based on the feature’s attributes. Use the `MGLVectorStyleLayer.predicate`
 property to include only the features in the source layer that satisfy a
 condition that you define.
 
+When a predicate is used for filtering vector data the key should be cast
+explicitly into the key's type. A number based key will become: `CAST(key, 'NSNumber')`
```

Comment:
Filtering vector data is the subject of this section, so it’s a bit redundant to say so.